packages/lesswrong/components/posts/TableOfContents/AnswerTocRow.jsx

```diff
@@ -27,6 +26,28 @@ const styles = (theme) => ({
     marginBottom: theme.spacing.unit*2,
     display:"flex",
     justifyContent: "space-between"
+  },
+  firstLine: {
+    fontSize: 13.2,
+    width: "calc(100% - 20px)",
+    fontFamily: theme.typography.postStyle.fontFamily,
+    marginTop: 0,
+    marginBottom: 0,
+    '& *': {
+      display: "inline"
+    },
+    '& blockquote, & br, & figure, & img': {
```

Comment:
Special casing HTML element types like this makes me uneasy; this seems likely to be a tarpit of special cases we haven't thought of. Perhaps we could start with something that's been through `html-to-text` on the server, rather than try to mangle HTML into shape?

Reply:
(Not an important bug, but an example of my sense of doominess around handling HTML in this context in general: On my local test install I asked "What is the answer to life, the universe, and everything?" and answered "42." The "42." rendered as "42." in the main answer box, but as "1." in the ToC, because Markdown did something weird with list-numbering.)

Reply:
Yeah I didn't know about html-to-text at the time I started this and agree that just using that usually makes more sense.

Only concern is that I specifically sometimes want to get rid of blockquotes (I guess the images and <br> tags get handled automatically?)

Reply:
Oh, I see where the usage is (it was in a format string and I somehow missed it).